extensions/amp-story/amp-story.md

```diff
@@ -934,13 +934,36 @@ To use it, include an `<amp-story-bookend>` tag as the child of your `<amp-story
   <amp-story-page id="cover">
     ...
   </amp-story-page>
-  <!-- `src` and `layout=display` are required. -->
+  <!-- `layout=display` is required. -->
   <amp-story-bookend src="bookendv1.json" layout=nodisplay>
   </amp-story-bookend>
 <amp-story>
 ```
 
-Next, you must create a JSON file where you can customize the bookend. The overall structure of the config looks like so:
+If you don't want to fetch the bookend config from a source, you can also specify it inline:
+
+```html
+<amp-story standalone>
+  <amp-story-page id="cover">
```

Comment:
Nit: maybe we can save a few lines by not repeating `amp-story-page` here, and have the `shareProviders` + `components` on one line each. It'd make sense if it's the second example